frontend/src/components/orgs-list.ts

```diff
@@ -93,6 +144,70 @@ export class OrgsList extends LiteElement {
     `;
   }
 
+  private renderOrgReadOnly() {
+    return html`
+      <btrix-dialog
+        class="[--width:36rem]"
+        id="orgReadOnlyDialog"
+        .label=${msg(str`Make Read-Only?`)}
+        @sl-after-hide=${() => (this.currOrg = null)}
+      >
+        ${when(this.currOrg, (org) => {
+          return html`
+            <p class="mb-3">
+              ${msg(
+                html`Are you sure you want to make
+                  <span class="font-semibold">${org.name}</span>
+                  read-only? All members' access will be reduced to the viewer
+                  role.`,
```

Comment:
Hm, not quite true anymore, as when an org is read-only we're now going to leave user permissions as they are and they'll still be able to see e.g. QA results.

Maybe "Are you sure you want to make ORG read-only? Members will no longer be able to crawl, upload files, create browser profiles, or create collections." ?